docs/ref/databases.txt

```diff
@@ -1107,6 +1107,46 @@ Example of a full DSN string::
         "(CONNECT_DATA=(SERVICE_NAME=orclpdb1)))"
     )
 
+.. _oracle-pool:
+
+Connection pool
```

Comment:
As far as I'm aware connection and session pooling are not the same thing. I think you copy too many things from PostgreSQL connection pooling PR. Please correct wording.

Reply:
@felixxm  oracledb documentation also mentions it as  connection pooling. For reference https://python-oracledb.readthedocs.io/en/latest/user_guide/connection_handling.html#connection-pooling . Should I replace connection with sessions?

Reply:
@suraj-ora-2020 I believe Mariusz is right, if you look into the docs you linked it says:

>In python-oracledb Thick mode, the pool implementation uses Oracle’s [**session pool** technology](https://www.oracle.com/pls/topic/lookup?ctx=dblatest&id=GUID-F9662FFB-EAEF-495C-96FC-49C6D1D9625C)

And then if you go into the linked Oracle docs, session pooling and connection pooling are different.

Reply:
@sarahboyce  drivers like the python driver (**oracledb**) or node.js driver refer to a connection+session combination as a connection. And OCI's session pooling is nothing but the same, a session from a session pool actually means  session+connection.

Reply:
I agree with @suraj-ora-2020 : drivers like python-oracledb and node-oracledb deliberately use 'connection pool', so that terminology should also be used in Django and frameworks that use the Oracle driver.